CVS-71745 update linux installation guide (#8979)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-linux.md

# Install and Configure Intel® Distribution of OpenVINO™ toolkit for Linux* {#openvino_docs_install_guides_installing_openvino_linux}

## Introduction

By default, the [OpenVINO™ Toolkit](https://docs.openvinotoolkit.org/latest/index.html) installation on this page installs the following components:

| Component                                                                                           | Description                                                                                                                                                                                                                                                                                                   |  
|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | [Inference Engine](../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications. |

Reviewer: Gr: It includes a set of libraries for an easy inference -> for easy inference

> | [Inference Engine](../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications. |

Reviewer: Gr: It includes a set of libraries for an easy inference -> for easy inference

| [Inference Engine Code Samples](../IE_DG/Samples_Overview.md) | A set of simple command-line applications demonstrating how to utilize specific OpenVINO capabilities in an application and how to perform specific tasks, such as loading a model, running inference, querying specific device capabilities, and more. |

> | [Demo Applications](@ref omz_demos) | A set of command-line applications that serve as robust templates to help you implement multi-stage pipelines and specific deep learning scenarios. |

Reviewer 2: should be removed

> > **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, OpenCV, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are available only on [pypi.org](https://pypi.org/project/openvino-dev/) now.

Reviewer: These tools are available only on [pypi.org](https://pypi.org/project/openvino-dev/) now. -> These tools are now only available on [pypi.org](...)

| Component                                                                                           | Description                                                                                                                                                                                                                                                                                                   |  
|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [Inference Engine](../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications. |

> | Intel® Media SDK | Offers access to hardware accelerated video codecs and frame processing |

Reviewer 2: Intel® Media SDK is EOL in 21.4 and will be removed. Please remove this row.

| [Inference Engine Code Samples](../IE_DG/Samples_Overview.md) | A set of simple command-line applications demonstrating how to utilize specific OpenVINO capabilities in an application and how to perform specific tasks, such as loading a model, running inference, querying specific device capabilities, and more. |
| [Demo Applications](@ref omz_demos) | A set of command-line applications that serve as robust templates to help you implement multi-stage pipelines and specific deep learning scenarios. |

> | [Documentation for Pre-Trained Models ](@ref omz_models_group_intel) | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo). |

Reviewer 2: OMZ demos & Documentation for Pre-Trained Model are also out of scope installer content.
@vladimir-dudnik, Do we need a reference or instructions on how to obtain this OMZ content in install guide?

- Ubuntu 18.04.x long-term support (LTS), 64-bit
- Ubuntu 20.04.0 long-term support (LTS), 64-bit

> > **NOTE**: Since the OpenVINO™ 2022.1 release, CentOS 7.6, 64-bit is not supported.

Reviewer: Gr: **NOTE**: Since the OpenVINO™ 2022.1 release, CentOS 7.6, 64-bit is not supported. -> is no longer supported

## Overview

> > **NOTE**: These steps apply to Ubuntu\* and with some modifications shown below, also Red Hat\* Enterprise Linux\*.

Reviewer 2: I believe this note is out of date, we should remove it

   Select the Intel® Distribution of OpenVINO™ toolkit for Linux package from the dropdown menu.

2. Open a command prompt terminal window. You can use the keyboard shortcut: Ctrl+Alt+T
3. Change directories to where you downloaded the Intel Distribution of OpenVINO toolkit for Linux\* file.<br>
   If you downloaded the starter script to the current user's `Downloads` directory:
   ```sh
   cd ~/Downloads/
   ```
   You should find there a bootstrapper script `l_openvino_toolkit_p_<version>.sh`.
4. Add executable rights for the current user:
   ```sh
   chmod +x l_openvino_toolkit_p_<version>.sh
   ```
5. If you want to use graphical user interface (GUI) installation wizard, run the script without any parameters:
   ```sh
   ./l_openvino_toolkit_p_<version>.sh
   ```
   <br>
   
   |![](../img/openvino-install-linux-installer-1.png)|
   |---|
   
   <br>Otherwise, you can add parameters `-a` for additional arguments and `--cli` to run installation in command line (CLI):
   ```sh
   ./l_openvino_toolkit_p_<version>.sh -a --cli
   ```

>    As you can see, there's no need for `sudo` command to be used.

Reviewer 2: storytelling style, not a guidance

> **NOTE**: To get additional information on all parameters that can be used, check up the help option: `--help`. Among others, you can find there `-s` option which offers silent mode, which together with `--eula approve` allows you to run whole installation with default values without any user inference.

7. Follow the instructions on your screen. During the installation you will be asked to accept the license agreement. The acceptance is required to continue.<br> 

   |![](../img/openvino-install-linux-run-boostrapper-script.gif)|

>    |---|

Reviewer 2: formatting

10. The following screen will present you with any libraries and packages that are required for successful OpenVINO installation. Click **Install** button and all missing dependencies will be installed on your machine. Once all installations are ready, click **Finish** button to go to **Installed Products** screen.

11. The **Finish** screen indicates that the OpenVINO toolkit has been installed.
   You can now close the installation wizard with exit button in the top right-hand corner of the screen or **Finish** button. However, you can also choose from the following options:
    * **Get Started Guide** - will redirect you to our documentation site in a browser.
    * **Modify** - will offer 4 more options:
      *  **Add/Remove components** - will give you a possibility of installing and uninstalling components.
      *  **Download a customized offline package** - allows you to download a package with components of your selection.
      *  **Repair** - will reinstall the OpenVino toolkit after confirmation.

>       *  **Uninstall** - will let you uninstall the OpenVino toolkit after confirmation. This action cannot be reversed.

Reviewer 2: I believe this should not be part of this document. Instead of there it would be better to suggest user to follow Getting Started Guide, Release notes or open install directory by link.

- Ubuntu 18.04.x long-term support (LTS), 64-bit
- Ubuntu 20.04.0 long-term support (LTS), 64-bit

> - For deployment on Red Hat* Enterprise Linux* 8.2, 64-bit, you can use the Intel® Distribution of OpenVINO™ toolkit runtime package that includes the Inference Engine core libraries, nGraph, Python bindings, and CPU and GPU plugins. The package is available as: 

Reviewer 2: Starting 2022.1 Red Hat* Enterprise Linux* 8.2 will be part of installer besides other distribution: archive, PyPI, Docker Image. I think this clarification should be removed.

   - [Downloadable archive](https://storage.openvinotoolkit.org/repositories/openvino/packages/2021.4.1/l_openvino_toolkit_runtime_rhel8_p_2021.4.689.tgz)
   - [PyPi package](https://pypi.org/project/openvino/)

>    - [Docker image](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)

Reviewer 2: Starting 2022.1 Red Hat* Enterprise Linux* 8.2 will be part of installer besides other distribution: archive, PyPI, Docker Image. I think this clarification should be removed.

4. Add executable rights for the current user:
   ```sh
   chmod +x l_openvino_toolkit_p_<version>.sh
   ```
5. If you want to use graphical user interface (GUI) installation wizard, run the script without any parameters:
   ```sh
   ./l_openvino_toolkit_p_<version>.sh
   ```
   <br>
   
   ![](../img/openvino-install-linux-installer-1.png)
   
   <br>
   <br>Otherwise, you can add parameters `-a` for additional arguments and `--cli` to run installation in command line (CLI):
   ```sh
   ./l_openvino_toolkit_p_<version>.sh -a --cli
   ```

   > **NOTE**: To get additional information on all parameters that can be used, check up the help option: `--help`. Among others, you can find there `-s` option which offers silent mode, which together with `--eula approve` allows you to run whole installation with default values without any user inference.

6. Follow the instructions on your screen. During the installation you will be asked to accept the license agreement. The acceptance is required to continue. Check out the installation process on the image below:<br>

   ![](../img/openvino-install-linux-run-boostrapper-script.gif)

   <br>

>    <br>By default, the Intel® Distribution of OpenVINO™ is installed to the following directory:

Reviewer 2: ```suggestion
   <br>By default, the Intel® Distribution of OpenVINO™ is installed to the following directory:
   
```

Author: I don't see the difference

   * For root or administrator: `/opt/intel/openvino_<version>/`
   * For regular users: `/home/<USER>/intel/openvino_<version>/`

   <br>For simplicity, a symbolic link to the latest installation is also created: `/opt/intel/openvino_2022/` or `/home/<USER>/intel/openvino_2022/`

> 8. **Optional**: You can choose **Customize** to choose the components you want to install. You can choose between Inference Engine and Deep Learning Workbench, however you have to choose at least one of them.

Reviewer 2: ```suggestion
8. **Optional**: You can choose **Customize** to choose the components you want to install. You can select Inference Engine Plugins for your target devices. You can find more information on supported derives at the [page](@ref openvino_docs_IE_DG_supported_plugins_Supported_Devices).
```

>    > **NOTE**: From pull-down menu you can choose which Inference Engine version you want to install with regard to the hardware.  

Reviewer 2: ```suggestion
```

   ```sh
   cd ~/Downloads/
   ```
   You should find a bootstrapper script [name needed].
4. Run the script:
   ```sh
   [command needed]
   ``` 
   <!--- By default, the file is saved as `l_openvino_toolkit_p_<version>.tgz`, e.g., `l_openvino_toolkit_p_2021.4.689.tgz`. --->
4. Unpack the .tgz file:
   ```sh
   tar -xvzf l_openvino_toolkit_p_<version>.tgz
   ```

>    The files are unpacked to the `l_openvino_toolkit_p_<version>` directory. 

Reviewer 2: not relevant any more. installer is provided as a bootstrapper script

After configuration is done, you are ready to run the verification scripts with the HDDL Plugin for your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs. Check up our [Movidius VPU demos](movidius-demos.md).

> **IMPORTANT**: While working with either HDDL or NCS, choose one of them as they cannot run simultaneously on the same machine.

## <a name="get-started"></a>Step 6: Start Using the Toolkit

Now you are ready to try out the toolkit. To continue, see the [Get Started Guide](../get_started/get_started.md) section to learn the basic OpenVINO™ toolkit workflow and run [code samples](../IE_DG/Samples_Overview.md) in C, C++, and Python and [notebook tutorials](../tutorials.md) that we prepared for you.

## Troubleshooting

PRC developers might encounter pip errors during OpenVINO™ installation. To resolve the issues, try one of the following options:
* Add the download source using the `-i` parameter with the Python `pip` command. For example: 

   ```

>    pip install numpy -i https://mirrors.aliyun.com/pypi/simple/

Reviewer 2: ```suggestion
   pip install openvino-dev -i https://mirrors.aliyun.com/pypi/simple/
```

The environment variables are set. Next, you will configure the Model Optimizer.

## <a name="model-optimizer">Step 4: Download additional components (Optional)

> > **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, OpenCV, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are available only on [pypi.org](https://pypi.org/project/openvino-dev/) now.

Reviewer: Gr: These tools are available only on pypi.org now. -> These tools are now only available on pypi.org

> Among developer tools you can find e.g. Model Optimizer which imports, converts, and optimizes models, and Model Downloader which gives you access to the collection of pre-trained deep learning public and intel-trained models.  

Reviewer: Gr: Among developer tools -> Among the developer tools

   [install]
   trusted-host = mirrors.aliyun.com
   ```

> 

Reviewer 2: Installation does not suppose this scenario.
